optuna/integration/keras.py

```diff
@@ -57,6 +58,12 @@ def on_epoch_end(self, epoch: int, logs: Optional[Dict[str, float]] = None) -> N
         logs = logs or {}
         current_score = logs.get(self._monitor)
         if current_score is None:
+            message = (
+                "The metric '{}' is not in the evaluation logs for pruning. "
+                "'{}' may not exist as evaluation metric for pruning. "
+                "Or '{}' may be a typo.".format(self._monitor, self._monitor, self._monitor)
```

Comment:
IMO, this message is a bit verbose because `self._monitor` appears three times in it. Can we simplify it, for example, as follows:

> "The metric '{}' is not in the evaluation logs for pruning. Please make sure you set the correct metric name."

Reply:
If brevity is the priority, that seems better. I will revise the message that way.